src/mscorlib/src/System/Runtime/CompilerServices/YieldAwaitable.cs

```diff
@@ -100,11 +100,11 @@ private static void QueueContinuation(Action continuation, bool flowContext)
                     {
                         if (flowContext)
                         {
-                            ThreadPool.QueueUserWorkItem(s_waitCallbackRunAction, continuation);
+                            ThreadPool.QueueUserWorkItem(s_waitCallbackRunAction, continuation, preferLocal: true);
                         }
                         else
                         {
-                            ThreadPool.UnsafeQueueUserWorkItem(s_waitCallbackRunAction, continuation);
+                            ThreadPool.UnsafeQueueUserWorkItem(s_waitCallbackRunAction, continuation, preferLocal: true);
```

Comment:
How do you know these are safe to change / that some use of `await Task.Yield()` isn't relying on the fairness?  Seems suspect to me; you could easily come up with a case where the change negatively impacted behavior.  And if these did change, it's creating a discrepancy from the else block below, where `PreferFairness` is specified explicitly.

Reply:
It was mainly `Task.Yield()` I was interested in - however; it would be a behaviour change 😦